src/functions/getValueFromTheme.js

```diff
@@ -0,0 +1,8 @@
+import { get } from 'lodash';
+import { theme } from '../components/publiq-ui/theme';
+
+const getValueFromTheme = (path) => {
+  return get(theme, path);
+};
+
+export { getValueFromTheme };
```

Comment:
Don't really like the idea of a 'functions' folder, looks super weird to me. I prefer to keep those as close to where we're using them. So in this scenario, I would just keep it in the `publiq-ui` folder, which also makes a lot more sense if we want to distribute it later.

Reply:
This is also not how the theming works, you have to retrieve it from the props in the styled-component.